TicketBooking.Tests/Flights/Services/FlightServicesTests.cs

```diff
@@ -0,0 +1,172 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using Moq;
+using TicketBooking.Airports;
+using TicketBooking.Classes;
+using TicketBooking.Countries;
+using TicketBooking.Flights.Models;
+using TicketBooking.Flights.Repository;
+
+namespace TicketBooking.Flights.Services;
+
+public class FlightServicesTests
+{
+    private readonly FlightServices _flightService;
+    private readonly Mock<IFlightRepository> _flightRepositoryMock;
+    private readonly Mock<List<Flight>> _flightsMock;
+    private readonly Mock<IFlightPrinter> _flightPrinterMock;
+    private readonly Fixture _fixture;
+
+    public FlightServicesTests()
+    {
+        _flightRepositoryMock = new Mock<IFlightRepository>();
+        _flightsMock = new Mock<List<Flight>>();
+        _flightPrinterMock = new Mock<IFlightPrinter>();
+        _flightService = new(_flightsMock.Object, _flightRepositoryMock.Object, _flightPrinterMock.Object);
+        _fixture = new Fixture();
+    }
+
+    [Fact]
+    public void LoadFlights_ShouldCallRepositoryMethod()
```

Comment:
Usually we do like what you did in `FilterFlights_ShouldReturnFlights_FiltersMatch`. We create a list and let the repository function returns the same list.

Reply:
Please also update the name to be more precise

Reply:
But tested method is void, it doesn't return any value